tcms/issuetracker/kiwitcms.py

```diff
@@ -34,7 +34,12 @@ def details(self, url):
             Provide more details from our own bug tracker!
         """
         bug_id = self.bug_id_from_url(url)
-        bug = Bug.objects.get(pk=bug_id)
+        bug = Bug.objects.filter(pk=bug_id)
```

Comment:
filter().first() will return object or none.

Reply:
The change from this commit, together with the previous one needs tests and can be independently pushed to master.